portfolio/src/main/webapp-local/src/App.js

```diff
@@ -3,7 +3,7 @@ import About from './components/About';
 import HomePage from './components/HomePage';
 import Projects from './components/Projects';
 import Contacts from './components/Contacts';
-import {BrowserRouter as Router} from 'react-router-dom';
+// import {BrowserRouter as Router} from 'react-router-dom';
```

Comment:
I know commenting out lines instead of deleting them is a pretty common habit, but as long as you're committing early and often you can feel free to just delete things entirely. If you ever want to undelete a line, you can go back in the git history to find it. (I can show you how to do this if you want!)

Reply:
Oops! rookie mistake! Don't wanna confuse anyone with those. Removing now :)